caveclient/frameworkclient.py

```diff
@@ -98,11 +98,10 @@ def __new__(
         version:
             The default materialization version of the datastack to use. If None, the
             latest version is used. Optional, defaults to None.
+        write_local_auth: bool, optional
```

Comment:
this appears to have been added to the doc string but this variable is not in the function signature or anywhere in the code.